### PR TITLE
Update Deathstreams

### PR DESCRIPTION
**Pull request by tematikkp**
Updated

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmapping/Mapping_techniques/Rhythm/en.md [hunk 1]
```diff
@@ -165,8 +165,7 @@ These consist of sliders (with or without repeats), hit circles or a combination
 
 The job of a deathstream is to practically fail players that cannot handle ordinary streams.
 
-The use of deathstreams will make your map **unrankable**!
-However, it may go through as one of the *approved* beatmaps.
+The use of deathstreams used to make your map unrankable, but now it is rankable, as approved category no longer exists.
```
Comment (Nivalyx) [inline wiki/Beatmapping/Mapping_techniques/Rhythm/en.md:168]: "deathstreams" have never been unrankable/strictly disallowed by the Ranking Criteria in practice tho

it's just that back then when osu! was still in its Score Ranking era (and Ranked beatmaps were prohibited from having 20mil+ max score), the usage of super long streams were rather discouraged as they tend to crank up the max score rapidly

Comment (tematikkp) [reply wiki/Beatmapping/Mapping_techniques/Rhythm/en.md:168]: Yeah, I just want to update this, as deathstreams are common in ranked maps.

Comment (Nivalyx) [reply wiki/Beatmapping/Mapping_techniques/Rhythm/en.md:168]: might wanna word this as such

```suggestion
During the [score ranking](/wiki/Ranking#score-ranking) era, the usage of deathstreams were strictly limited as the [Ranking Criteria](/wiki/Ranking_Criteria) would only allow for a Ranked beatmap to have a maximum of 20 million Ranked score back then.
```

File: wiki/Beatmapping/Mapping_techniques/Rhythm/en.md [hunk 2]
```diff
@@ -165,8 +165,7 @@ These consist of sliders (with or without repeats), hit circles or a combination
 
 The job of a deathstream is to practically fail players that cannot handle ordinary streams.
 
-The use of deathstreams will make your map **unrankable**!
-However, it may go through as one of the *approved* beatmaps.
+During the [score ranking](/wiki/Ranking#score-ranking) era, the use of deathstreams were strictly limited as the [Ranking Criteria](/wiki/Ranking_Criteria) would only allow for a Ranked beatmap to have a maximum score of 20 million. However, with the introduction of the [pp system](http://osu.ppy.sh/wiki/en/Performance_points), the score cap rule was soon rendered moot and deathstreams saw a tick up in its usage as a result.
```
Comment (Pisapou) [inline wiki/Beatmapping/Mapping_techniques/Rhythm/en.md:168]:
```suggestion
During the [score ranking](/wiki/Ranking#score-ranking) era, the use of deathstreams were strictly limited as the [Ranking Criteria](/wiki/Ranking_Criteria) would only allow for a Ranked beatmap to have a maximum score of 20 million. However, with the introduction of the [pp system](/wiki/en/Performance_points), the score cap rule was soon rendered moot and deathstreams saw a tick up in its usage as a result.
```

Comment (Pisapou) [inline wiki/Beatmapping/Mapping_techniques/Rhythm/en.md:168]:
```suggestion
During the [score ranking](/wiki/Ranking#score-ranking) era, the use of deathstreams were strictly limited as the [Ranking Criteria](/wiki/Ranking_Criteria) would only allow for a Ranked beatmap to have a maximum score of 20 million. However, with the introduction of the [pp system](/wiki/en/Performance_points), the score cap rule was soon rendered moot and deathstreams saw a tick up in its usage as a result.
```

**Review by Pisapou — CHANGES_REQUESTED** [review]

**Review by TicClick — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**